src/index.js

```diff
@@ -0,0 +1,49 @@
+import './styles.css';
+
+const toDoArr = [
+  {
+    description: 'wash the dishes',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'fix the car',
+    completed: false,
+    index: 0,
+  },
+  {
+    description: 'walking the dog',
+    completed: false,
+    index: 0,
+  }];
```

Comment:
## [OPTIONAL]

- [ ] Kindly make sure your indexes are unique as this should uniquely identify the to-do task.